extension/new-tab.js

```diff
@@ -5,11 +5,18 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
+
+// TODO(smalls) - there should be a better system of unique ids
+var faux_gid = 2000;
+
 /**
  * Load the current browsing data from all non-https tabs on all devices.
```

Comment:
what is the non-https restriction here?

Reply:
Chalk this up to me mis-understanding a regex below. I'll remove this in the future (just a comment saying 'loads browsing data' isn't particularly helpful).